app/components/node/connect/node-connect.scss

```diff
@@ -1,50 +1,171 @@
+            align-items: center;
         }
 
-        &:active {
-            color: $whitesmoke;
-            background-color: map-get($primary, 500);
+        .node-information {
+            bl-property-group {
+                margin: 15px 0;
+
+                .custom-property {
+                    display: flex;
+                    padding: 0;
+                    margin: 0 0 5px;
+                    position: relative;
+                    width: 100%;
+                    overflow: visible;
+
+                    .label-small {
+                        color: $secondary-text;
+                        font-size: 13px;
+                        flex-basis: 200px;
+                        font-weight: bold;
+                        line-height: 20px;
+                        margin-right: 5px;
+                    }
+
+                    > input {
+                        flex: 1;
+                        max-width: 349px;
+                    }
+
+                    > .display-field {
+                        flex: 1;
+                        max-width: 350px;
+                        background-color: $main-background;
+                        padding: 3px 8px;
+                        margin: 0;
+                        text-overflow: ellipsis;
+                        overflow: hidden;
+                        display: block;
+                        width: 100%;
+                        min-height: 24px;
+                    }
+
+                    bl-clickable {
+                        color: $link-color;
+                    }
+
+                    .password-failure-hint {
+                        position: absolute;
+                        top: 27px;
+                        left: 205px;
+                        color: red;
+                    }
+
+                    .clipboard-info {
+                        position: absolute;
+                        top: 28px;
+                        left: 210px;
+                        color: $secondary-text;
+                    }
+                }
+            }
         }
-    }
-}
 
-bl-node-user-credentials-form {
-    display: block;
-    margin-bottom: 20px;
+        .credentials-inputs {
+            width: 400px;
+            display: flex;
+            align-items: center;
+            margin: 10px;
+
+            > input {
+                width: 200px;
+                margin: 5px;
+                padding: 5px;
+            }
+        }
+
+        .credentials-source {
+            display: flex;
+            justify-content: center;
+
+            > bl-button {
+                user-select: none;
+                width: 150px;
+                height: 150px;
+                font-size: 20px;
+                margin: 10px 20px;
+                text-align: center;
+                cursor: pointer;
+                position: relative;
+
+                &:hover {
+                    color: $whitesmoke;
+                    background-color: map-get($primary, 400);
```

Comment:
try not to use the `map-get($primary,` use `$primary-color`  or `$primary-color-light/dark`